google/cloud/spanner_v1/batch.py

```diff
@@ -183,11 +189,17 @@ def commit(self, return_commit_stats=False, request_options=None):
         # Request tags are not supported for commit requests.
         request_options.request_tag = None
 
+        max_commit_delay = None
+        if max_commit_delay_ms:
+            max_commit_delay = Duration()
+            max_commit_delay.FromMilliseconds(max_commit_delay_ms)
+
         request = CommitRequest(
             session=self._session.name,
             mutations=self._mutations,
             single_use_transaction=txn_options,
             return_commit_stats=return_commit_stats,
+            max_commit_delay=max_commit_delay,
```

Comment:
What are the validations that backend does on this `max_commit_delay` field?
For ex, does backend accept only data in `ms` etc.

Reply:
Any Duration is fine to pass in here. If the duration is outside of the allowed range [0, 500] the backend will complain and pass an error back out. 

I would suggest we don't implement any bounds checks here - I think it's fine to allow the backend to return the error, and then we can have all the bounds checking in one place. If we ever increase the range, I'd like to not have to make client changes.